src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java

```diff
@@ -1124,6 +1287,149 @@ protected Name readSystemPropertyName() {
         return names.fromChars(buf, pos, bp - pos);
     }
 
+    protected int readIndent() {
+        int indent = 0;
+        while (bp < buflen) {
+            switch (ch) {
+                case ' ' -> indent++;
+                case '\t' -> indent = 4;
```

Comment:
I did mean `indent = 4`.
It would not mean `indent +=4` because you would have to take preceding character count into account, to round up to a multiple of 4.
But anyway, it's enough to know the indent is 4 or more, meaning the code is looking at an indented code block.
https://spec.commonmark.org/0.30/#indented-code-blocks